__tests__/endpoint.tests.js

```diff
@@ -73,6 +73,7 @@ describe('GET/api/article/:article_id', () => {
             body: "I find this existence challenging",
             created_at: new Date(1594329060000).toISOString(),
             votes: 100,
+            comment_count: 11,
```

Comment:
we should create a new test for this, rather than updating a previous test - what would be a better matcher to use in this case than toEqual that would still allow this to work even if we add a new property to what's being returned here?